app/controllers/public/index.js

```diff
@@ -112,8 +112,13 @@ export default Controller.extend({
         this.set('isLoginModalOpen', true);
         return;
       }
-      let order = this.get('model.order');
-      let event = order.get('event');
+      let order = await this.get('model.order');
+      let event = await order.get('event');
+      let tax = this.get('model.event.tax');
```

Comment:
@pradeepgangwar I removed `await` as we are including `tax` while fetching the event model.

Reply:
Also, it's working fine with this.